src/GraphQl/Queries/Queries.ts

```diff
@@ -336,8 +336,9 @@ export const MEMBERSHIP_REQUEST = gql`
         user {
           _id
           firstName
-          lastName
           email
+          image
+          createdAt
```

Comment:
Why have you removed lastName from the query here? It might break the talawa app if any component is depending upon lastName field.